src/lib/regression.ml

```diff
@@ -63,7 +63,7 @@ module Univariate = struct
     ; inferred_var     : float          (* inferred variance of error. *)
     ; s_yy             : float          (* sum of diff or resp to mean, TSS. *)
     ; s_xx             : float          (* sum of diff of pred to mean. *)
-    ; goodness_of_fit  : float option
+    (*; goodness_of_fit  : float option *)
```

Comment:
No particular good reason, laziness I guess. I already filed an issue #130 that should address this.